app/services/coms.js

```diff
@@ -395,6 +395,11 @@ export default class ComsService extends Service {
   handleSockethubMessage (message) {
     this.log(`${message.context}_message`, 'SH message', message);
 
+    if (message.actor.type === 'service') {
+      this.log(`${message.context}_message`, 'skipping service message');
+      return;
+    }
```

Comment:
Service update messages are already handled in the `update` case below. Are there any new message types from the service that we should handle as well?

Reply:
Well as the schema is defined now, it's not required. The only thing required is `id` and `type`.

Reply:
Yeah I think this section should be improved, my skipping isn't a good solution. The handling further down of the `service` type is only in the case of `update`s, however sometimes there'll be a `send` from the `service`. Hyperchannel was erroring trying to display it like a person writing to a room. 

So in this case, it's a message from the service, with no room. We should either display those some other way or just ignore them or write them to the console. The `send` case should verify first that the target is a `room` before calling `addMessageToChannel`